backend/bootstrap.py

```diff
@@ -3,19 +3,20 @@
 from sqlmodel import select, func
 from sqlmodel.ext.asyncio.session import AsyncSession
 from colorama import Fore, Style
+import click
```

Comment:
Hi, we should add it to pyproject.toml via poetry?

Reply:
Yes

Reply:
@chethanuk Yes, I mean could you help submit click to pyproject 😂